server/app/data.yml

```
@@ -80,6 +80,7 @@ defaults:
     uploads:
       maxFileSize: 5242880
       maxFiles: 10
+    bodyParserSize: 1
```

What do you think to increase this value because of bigger diagrams? Something like this:
```suggestion
    bodyParserSize: 20
```

based on pr: https://github.com/Requarks/wiki/pull/3850#pullrequestreview-848083347